dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassDumperTest.java

```diff
@@ -43,7 +43,7 @@
 import org.junit.Test;
 
 public class ClassDumperTest {
-  private static final String EXAMPLE_JAR_FILE =
+  static final String EXAMPLE_JAR_FILE =
```

Comment:
Rather than making this nonprivate, and coupling your tests, I think you should copy it to the other test that needs it.

Reply:
Copied.